website/static/js/accountSettings.js

```diff
@@ -193,6 +209,21 @@ var UserProfileViewModel = oop.extend(ChangeMessageMixin, {
             this.changeMessage('Email cannot be empty.', 'text-danger');
         }
     },
+    reSendConfirmation: function(email){
+        var self = this;
+        self.changeMessage('', 'text-info');
+        bootbox.confirm({
+            title: 'Resend Email Confirmation?',
+            message: 'Are you sure that you want to resend email confirmation at ' + '<em><b>' + email.address() + '</b></em>',
+            callback: function (confirmed) {
+                if (confirmed) {
+                    self.client.update(self.profile(), email).done(function () {
+                        $osf.growl('Email confirmation resent to ', '<em>' + email.address() + '<em>', 'success');
```

Comment:
fixed in 6e6e6fe.
Now:
![screen shot 2015-06-11 at 11 01 16 am](https://cloud.githubusercontent.com/assets/4974056/8110254/3d51a34e-1029-11e5-8dfd-ec106ed545df.png)